schema.graphql

```diff
@@ -266,11 +266,21 @@ input PlaylistAppendInput {
 }
 
 type Mutation {
-  # Increments the play count and updates the last played time in SongUserStats.
-  # Returns the songs stats.
-  playSong(songId: ID!): SongUserStats!
-
-  # Toggles the like state of the specified song. Returns the songs stats.
+  # Increments play counts and updates the last played time in SongUserStats.
+  # Returns the song's stats. Optionally one of artistId, albumId or playlistId,
+  # can be specified for calculating play counts of albums, artists and
+  # playlists. Specifing more than one of artistId, albumId or playlistId is an
+  # error.
+  playSong(
```

Comment:
The comment should specify that the reason for the optional parameters is so that the server can keep track of what context a song is played in.